tests/Jwt/FirebaseParserTest.php

```diff
@@ -53,30 +53,16 @@ protected function setUp()
 
     public function testParseTokenWithExpiredToken()
     {
-        $parser = new FirebaseParser($this->config);
+        $token = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJ1cmkiLCJpYXQiOiIxNDQzNzE5MjM2IiwiZXhwIjoxNDQzNzIyODM2LCJzdWIiOiJ1c2VyLTEifQ.YquvWRtH72muTv1hCCKH4HBkBhhCm1wTG0MpnyC-pvQ';
 
-        try {
-            $parser->parseToken(
-                'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJ1cmkiLCJpYXQiOiIxNDQzNzE5MjM2IiwiZXhwIjoxNDQzNzIyODM2LCJzdWIiOiJ1c2VyLTEifQ.YquvWRtH72muTv1hCCKH4HBkBhhCm1wTG0MpnyC-pvQ'
-            );
-            $this->fail('Expected exception was not thrown');
-        } catch (InvalidException $e) {
-            $this->assertSame(InvalidException::CODE_TOKEN_EXPIRED, $e->getCode());
-        }
-    }
+        $this->setExpectedException(
+            InvalidException::class,
+            sprintf('Provided auth token `%s` is expired', $token),
```

Comment:
I'm not a huge fan of verifying exception messages, because they are not functionally important.